packages/components/snackbar/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@spark-ui/snackbar",
+  "version": "1.0.0",
+  "description": "Display brief, temporary notifications. Meant to be noticed without disrupting a user's experience or requiring an action to be taken.",
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "@spark-ui",
+    "react",
+    "component",
+    "accessible",
+    "accessibility",
+    "wai-aria",
+    "aria",
+    "a11y",
+    "snackbar"
+  ],
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
```

Comment:
Don't you want to add the `react-aria/spectrum` deps?

Reply:
Why not, I thought I would do it later, but you're right :+1: